### PR TITLE
[PM-5024] tax info component migration

### PR DESCRIPTION
**Pull request by vinith-kovan**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [X] Other
```

## Objective

Migrate the tax info component to use the Component Library

## Code changes

- tax-info.component.html - Updated the inputs and select options to use the equivalent components from the Component Library.
- tax-info.component.ts - Added a JSON representation of the country list to be passed to the HTML select options.

## Screenshots

Before migration:
upgrade plan
image
image

Create new organization
image

After migration:
upgrade plan
image
image

Create new organization
image

### CODE REVIEW

File: apps/web/src/app/billing/shared/tax-info.component.ts [hunk 1]
```diff
@@ -39,7 +43,259 @@ export class TaxInfoComponent {
     country: "US",
     includeTaxId: false,
   };
-
+  countryList: CountryList[] = [
+    { name: "-- Select --", value: "", disabled: false },
```
Comment (willmartian) [inline apps/web/src/app/billing/shared/tax-info.component.ts:47]: Nice change moving this into the TS file. 👍 

Makes it a bit easier to filter/manipulate/share this in the future if needed.

File: apps/web/src/app/billing/shared/tax-info.component.html [hunk 2]
```diff
@@ -1,365 +1,88 @@
...
-    <div class="form-group form-check">
-      <input
-        class="form-check-input"
-        id="addressIncludeTaxId"
-        name="addressIncludeTaxId"
-        type="checkbox"
-        [(ngModel)]="taxInfo.includeTaxId"
-      />
-      <label class="form-check-label" for="addressIncludeTaxId">{{ "includeVAT" | i18n }}</label>
+    <div class="tw-relative tw-mb-4">
```
Comment (willmartian) [inline apps/web/src/app/billing/shared/tax-info.component.html:41]: I don't think we need `tw-relative tw-mb-4` anymore.

**Review by willmartian — CHANGES_REQUESTED** [review]
Hey @vinith-kovan,

Rather that replacing the bootstrap `row` and `col-` utilities with their source styles, could you instead use CSS grid? 

- `row` would be `tw-grid tw-grid-cols-12 tw-gap-4`
- `col-3` would be `tw-col-span-3`, etc

This should preserve both the styles and the _semantic_ meaning of the styles. Let me know if you run into any issues with this approach.

**Review by willmartian — APPROVED** [review]
Changes LGTM 👍

**Review by amorask-bitwarden — DISMISSED** [review]

**Review by amorask-bitwarden — CHANGES_REQUESTED** [review]
Will re-review after conflicts are resolved.

**Review by cyprain-okeke — COMMENTED** [review]
Looks good but we have conflicts

**Review by amorask-bitwarden — DISMISSED** [review]

**Review by cyprain-okeke — APPROVED** [review]

**Pull request merged**